tests/test_csp_rendering.py

```diff
@@ -13,6 +13,13 @@
 
 from .base import IntegrationTestCase
 
+MIDDLEWARE_CSP_BEFORE = settings.MIDDLEWARE[:]
```

Comment:
Python lists have a `.copy()` method which does the same but maybe a bit clearer?

Reply:
Makes sense!